ibis_substrait/compiler/translate.py

```diff
@@ -583,6 +583,36 @@ def unbound_table(
     )
 
 
+@translate.register(ops.LocalTable)
+def local_table(
+    op: ops.LocalTable, expr: ir.TableExpr, _: SubstraitCompiler, **kwargs: Any
+) -> stalg.Rel:
+    def path_type_of(name : str | None):
+        if name is None:
+            raise ValueError("local table name is None")
+        if name.startswith("file://"):
+            return "uri_folder" if name.endswith("/") else "uri_file"
+        else:
+            return "uri_path_glob" if name.contains("*") else "uri_path"
```

Comment:
```suggestion
            return "uri_path_glob" if glob.has_magic(name) else "uri_path"
```
You'll need to `import glob` at the top of the file as well.